<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectCompilerWidget</class>
 <widget class="QWidget" name="ProjectCompilerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>784</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="6" column="0" colspan="3">
    <widget class="QWidget" name="widget" native="true"/>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QCheckBox" name="chkStaticLink">
     <property name="text">
      <string>Statically link libraries</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="cbCompilerSet"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Base compiler set:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Customize (apply to this project only):</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="CompileArgumentsWidget" name="tabOptions">
     <property name="documentMode">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QWidget" name="panelAddCharset" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QCheckBox" name="chkAddCharset">
        <property name="text">
         <string>Set Encoding for the executable:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbEncoding"/>
      </item>
      <item>
       <widget class="QComboBox" name="cbEncodingDetails"/>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CompileArgumentsWidget</class>
   <extends>QTabWidget</extends>
   <header location="global">widgets/compileargumentswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
